src/workloads/execution/ColumnstoreIndexes.yml

```diff
@@ -0,0 +1,467 @@
+  OperationCommand: *DuplicatedStringFieldProjectionQuery
+- OperationMetricsName: CountPerActionQuery
+  OperationName: RunCommand
+  OperationCommand: *CountPerActionQuery
+- OperationMetricsName: TotalTenantCreatedQuery
+  OperationName: RunCommand
+  OperationCommand: *TotalTenantCreatedQuery
+- OperationMetricsName: ActiveCloudUsersLastMonthQuery
+  OperationName: RunCommand
+  OperationCommand: *ActiveCloudUsersLastMonthQuery
+- OperationMetricsName: VersionsHistogramQuery
+  OperationName: RunCommand
+  OperationCommand: *VersionsHistogramQuery
+- OperationMetricsName: UniqueUsersPerMonthQuery
+  OperationName: RunCommand
+  OperationCommand: *UniqueUsersPerMonthQuery
+- OperationMetricsName: TopErrorsQuery
+  OperationName: RunCommand
+  OperationCommand: *TopErrorsQuery
+- OperationMetricsName: NumberOfChartsPerTypeQuery
+  OperationName: RunCommand
+  OperationCommand: *NumberOfChartsPerTypeQuery
+- OperationMetricsName: EventsByVersionQuery
+  OperationName: RunCommand
+  OperationCommand: *EventsByVersionQuery
+- OperationMetricsName: BrowserUsageByDistinctUserQuery
+  OperationName: RunCommand
+  OperationCommand: *BrowserUsageByDistinctUserQuery
+- OperationMetricsName: TopTenantsByNoOfEventsQuery
+  OperationName: RunCommand
+  OperationCommand: *TopTenantsByNoOfEventsQuery
+
+# Defines how many times each test operation is executed.
+TestRepeatCount: &TestRepeatCount 10
+
+Actors:
+# The 'WarmupCache1' actor warms up the buffer cache.
+- Name: WarmupCache1
+  Type: RunCommand
+  Database: *db
+  Thread: 1
+  Phases:
+  - Repeat: 2
+    Threads: 1
+    Database: *db
+    Operations: *TestOperations
+  - Phase: 1..4
+    Nop: true
+
+# The 'ColumnstoreScan' actor measures the perf with columnar indexes.
+- Name: ColumnstoreScan
+  Type: RunCommand
+  Database: *db
+  Phases:
+  - &Nop {Nop: true}
+  - Repeat: *TestRepeatCount
+    Database: *db
+    Operations: *TestOperations
+  - Phase: 2..4
+    Nop: true
+
+# The 'DropColumnstoreIndexes' actor drops columnstore indexes.
+- Name: DropColumnstoreIndex
+  Type: RunCommand
+  Database: *db
+  Phases:
+  - *Nop
+  - *Nop
+  - Repeat: 1
+    Threads: 1
+    Database: *db
+    Operations:
+      - OperationName: RunCommand
+        OperationCommand:
+          dropIndexes: *Collection
+          index: $**_columnstore
+  - *Nop
+  - *Nop
+
+# The 'WarmupCache2' actor warms up the buffer cache for a regular scan.
+- Name: WarmupCache2
+  Type: RunCommand
+  Database: *db
+  Phases:
+  - *Nop
+  - *Nop
+  - *Nop
+  - Repeat: 2
+    Threads: 1
+    Database: *db
+    Operations: *TestOperations
+  - *Nop
+
+# The 'CollScan' actor measures the perf without columnar indexes.
+- Name: CollScan
+  Type: RunCommand
+  Database: *db
+  Phases:
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - Repeat: *TestRepeatCount
+    Database: *db
+    Operations: *TestOperations
+
+AutoRun:
+- When:
+    mongodb_setup:
+      $eq:
+      - standalone-all-feature-flags
+    branch_name:
```

Comment:
This is to run on master only? Would $eq be simpler (and work)

Reply:
I think Yoonsoo meant master and release version > 6.0 here.
True that we want this on master now, but if we do $eq master here then we need to come back and update this after we have 6.1 and later branched.

Reply:
Exactly Ethan!